Classes/Controllers/PBGitHistoryController.m

```diff
@@ -628,6 +632,42 @@ - (NSArray *)menuItemsForPaths:(NSArray *)paths
 	return menuItems;
 }
 
+- (void)didDoubleClickCommit:(id)sender
+{
+	id refish = nil;
+	
+	// If they double-clicked the row with ref labels in we need to check if they clicked a ref label.
+	if ([commitList.tableColumns[commitList.clickedColumn].identifier isEqualToString:@"SubjectColumn"])
+	{
+		// Get the cell.
+		PBGitRevisionCell *cell = (PBGitRevisionCell*)[commitList preparedCellAtColumn:commitList.clickedColumn row:commitList.clickedRow];
```

Comment:
This is a minor thing, but ISTR seeing that pattern, and I know it will conflict with the tableview changes in #103 (which is why I didn't review it at first).

If that pattern is repeated, and you find a way to move all that "get clicked refish from cell" cruft hidden in a `[PBCommitList clickedRefish]`, that would be ♥️ .